low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -276,18 +276,19 @@ def _transmit_msg_on_exchange(self):
                                              routing_key=self._ack_routing_key,
                                              properties=msg_props,
                                              body=jsonMsg)
-
+                self._log_debug("_transmit_msg_on_exchange, Successfully Sent: %s" % jsonMsg)
```

Comment:
Why a different interface is used ? logger.debug() is not used, why ?

Reply:
@ujjwalpl if you see old legacy code is just being restructured, to avoid duplicating dumping alerts when debug log level used. Anyways can address it.

@shriya-deshmukh plz do the needful

Reply:
Done.